docs/src/pages/_app.page.tsx

```diff
@@ -86,11 +86,15 @@ function MyApp({ Component, pageProps }) {
 
       <div className={isHomepage ? `docs-home` : ''}>
         <ThemeProvider theme={baseTheme} colorMode={colorMode}>
-          <GlobalNav
-            rightLinks={RIGHT_NAV_LINKS as NavMenuItem[]}
-            leftLinks={LEFT_NAV_LINKS as NavMenuItem[]}
-            currentSite="UI Library"
-          />
+          {/* Remove this conditional post-launch */}
+          {process.env.NEXT_PUBLIC_SHOW_GLOBAL_NAV ? (
```

Comment:
What do you think if we add a feature flag like [`FF_REACT_NATIVE`](https://github.com/aws-amplify/amplify-ui/blob/34eb153560ff426c048086ee558d20834c206ed7/docs/src/utils/featureFlags.ts)?